native/themes/markdown-styles.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import { markdownStyle } from '../themes/colors';
+
+const unboundStyles = {
+  autolink: {
+    color: 'link',
+    textDecorationLine: 'underline',
+  },
+  noMargin: {
+    marginBottom: 0,
+    marginTop: 0,
+  },
+  paragraph: {
+    alignItems: 'flex-start',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'flex-start',
+  },
+  paragraphCenter: {
+    alignItems: 'flex-start',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    textAlign: 'center',
+  },
+  text: {
+    color: 'listForegroundLabel',
+    fontFamily: 'Arial',
+    fontSize: 18,
+  },
+  textRow: {
+    flexDirection: 'row',
+  },
+  view: {
+    alignSelf: 'stretch',
+  },
+};
+
+export type Style = $Exact<typeof unboundStyles>;
+
+export function style(theme: string) {
```

Comment:
1. Let's name this export something more specific. How about `getMarkdownStyles`
2. `theme: GlobalTheme`